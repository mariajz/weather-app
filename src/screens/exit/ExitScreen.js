import React from 'react';
import { Text } from 'react-native';
import { ExitContentWrapper } from './ExitScreen.style';

const ExitScreen = () => {
    return (
        <ExitContentWrapper>
            <Text testID="text">
                We have encountered an error while fetching weather details.
                Please try again after some time.
            </Text>
        </ExitContentWrapper>
    );
};

export default ExitScreen;
