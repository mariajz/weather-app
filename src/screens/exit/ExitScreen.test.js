import { render } from '@testing-library/react-native';
import React from 'react';
import ExitScreen from './ExitScreen';

describe('ExitScreen', () => {
    it('should render ExitScreen', () => {
        const container = render(<ExitScreen />);

        expect(container).toMatchSnapshot();
    });
});
