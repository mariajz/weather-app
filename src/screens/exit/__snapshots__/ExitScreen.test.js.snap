// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExitScreen should render ExitScreen 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flexBasis": 0,
      "flexGrow": 1,
      "flexShrink": 1,
      "justifyContent": "center",
      "marginBottom": 10,
      "marginLeft": 10,
      "marginRight": 10,
      "marginTop": 10,
    }
  }
>
  <Text
    testID="text"
  >
    We have encountered an error while fetching weather details. Please try again after some time.
  </Text>
</View>
`;
